CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 3.3.1 (2020-03-11)
+
+- Fixed the country field component to not take up extra space below the component. [#59](https://github.com/blackbaud/skyux-lookup/pull/59)
+- Fixed the country field component to properly recognize if it is within a phone field component. [#59](https://github.com/blackbaud/skyux-lookup/pull/59)
```

Comment:
I'd remove "properly" and say "...to recognize when it is" instead. Is it _placed_ within the phone field component? If yes, then I'd say "...to recognize when it is placed within the phone field component.

Reply:
Done